modules/apps/object/object-test/src/testFunctional/tests/tests/VisualizeObjectCollectionDisplay.testcase

```diff
@@ -30,13 +30,19 @@ definition {
 	}
 
 	@description = "LPS-133865 - Verify if the Object is displayed to be selected as Collection Provider on the Collection Display fragment"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test ObjectDisplayedToCollectionProdiver {
 		property portal.acceptance = "true";
 
```

Comment:
Please, add the property to skip the virtual instance.

Reply:
@rodrigocunhaa  Done!